imports/client/components/PuzzleModalForm.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import Alert from 'react-bootstrap/lib/Alert';
-import ControlLabel from 'react-bootstrap/lib/ControlLabel';
-import FormControl from 'react-bootstrap/lib/FormControl';
-import FormGroup from 'react-bootstrap/lib/FormGroup';
+import Alert from 'react-bootstrap/Alert';
+import FormControl, { FormControlProps } from 'react-bootstrap/FormControl';
+import FormGroup from 'react-bootstrap/FormGroup';
+import FormLabel from 'react-bootstrap/FormLabel';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/esm/Col';
```

Comment:
what's this `esm/` doing here? https://react-bootstrap.github.io/layout/grid/#col-props suggests it should just be `'react-bootstrap/Col'`

Reply:
Bah, oversight. VS Code was very determined to pull its imports from there instead of top-level. Pushed a fix.